packages/react-native-win32/src/Libraries/Components/Picker/Tests/PickerWin32Example.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
```

Comment:
Can we use facebooks PickerExample test page instead of writing a separate one?

Reply:
do you mean in RNTestList I will reference this file?

Reply:
IIRC Facebook already has an RNTester page for picker. It's included in RNTesterList for react-native-windows, but not sure if react-native-win32.

Related, 0.61 changes RNTester structure, and we're working on the merge right now. We would likely need to pretty immediately change any new examples to fit into the new structure.